central/policy/service/service_impl.go

```diff
@@ -717,10 +721,10 @@ func (s *serviceImpl) ExportPolicies(ctx context.Context, request *v1.ExportPoli
 	if err != nil {
 		return nil, err
 	}
-	errDetails := &v1.ExportPoliciesErrorList{}
+	errDetails := &v1.PolicyOperationErrorList{}
 	for _, missingIndex := range missingIndices {
 		policyID := request.PolicyIds[missingIndex]
-		errDetails.Errors = append(errDetails.Errors, &v1.ExportPolicyError{
+		errDetails.Errors = append(errDetails.Errors, &v1.PolicyOperationError{
```

Comment:
Not sure, but confirming this does not break UI? or Scripts? Backward compatible?

Reply:
I think the API gets the errors as one concatenated string, the same one repeated in both the `message` and `error` field of an error response.

In short, I don't think the UI's usage of the API can know or care what type of error it is. Consumers of the API through gRPC _may_ be affected by this, however.